packages/destination-actions/src/destinations/google-enhanced-conversions/functions.ts

```
@@ -587,26 +601,7 @@ const addOperations = async (
     return response.data
   } catch (error) {
     statsContext?.statsClient?.incr('error.addOperations', 1, statsContext?.tags)
-
-    // Google throws 400 error for CONCURRENT_MODIFICATION error which is a retryable error
-    // We rewrite this error to a 500 so that Centrifuge can retry the request
-    for (const errorDetails of (error as GoogleAdsError).response?.data?.data?.error?.details) {
-      for (const errorItem of errorDetails.errors) {
-        if (errorItem?.errorCode?.databaseError) {
-          throw new RetryableError(
-            errorItem?.message ??
-              'Multiple requests were attempting to modify the same resource at once. Retry the request.',
-            500
-          )
-        }
-      }
-
-      throw new IntegrationError(
-        (error as GoogleAdsError).response?.statusText,
-        'INVALID_RESPONSE',
-        (error as GoogleAdsError).response?.status
-      )
-    }
```

This INVALID Response is actually not useful. So, I removed it let the error bubble through in     handleGoogleAdsError(error)